### PR TITLE
Changing the node label to use the latest jenkins worker

### PR DESCRIPTION
**Pull request by tdaines42**

### CODE REVIEW

**Comment by kiall** [comment]
I updated the 2x `-test` worker types to use the new / larger 'jenkins.half-dedicated' flavour - as the original "large" was too small, and we were seeing failures etc. I've restarted to PR CI on this.

File: vars/coreKubicProjectCi.groovy [hunk 1]
```diff
@@ -29,7 +29,7 @@ def call() {
     ])
 
     withKubicEnvironment(
-            nodeLabel: 'leap42.3&&caasp-pr-worker',
+            nodeLabel: 'prvcld-leap-15.0-test',
```
Comment (kiall) [inline vars/coreKubicProjectCi.groovy]: The "final" value here should be:

    nodeLabel: 'leap15.0&&caasp-pr-worker',

don't merge until it's that :)

Comment (tdaines42) [reply vars/coreKubicProjectCi.groovy]: I updated it to the correct one and everything seems to be working.

**Comment by tdaines42** [comment]
[(2) Leap 15 Jenkins worker image](https://trello.com/c/X7MXSXw3/1111-2-leap-15-jenkins-worker-image)

**Pull request merged**